### PR TITLE
Xamarin UWP project added

### PR DESCRIPTION
**Pull request by amoraitis**
Added uwp project and configured to work(beta)

- [ ] Add a new hockeyAppId(yours)

- [ ] In Mainpage.xaml.cs we must make the method LoadApplication to be able to run the project

- [ ] change publisher name(in manifest)

### CODE REVIEW

File: src/UWP/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage
+    {
+        private const string HockeyAppId = "5628855c469d43d2aa8e520664a45036";
+        public MainPage()
+        {
+            this.InitializeComponent();
+           
+            LoadApplication(new Bit.App.App(
```
Comment (kspearrin) [inline src/UWP/MainPage.xaml.cs]: Does this compile?

Comment (amoraitis) [inline src/UWP/MainPage.xaml.cs]: image
What to you mean. It doesnt compile we must add correct parameters(I wrote it on pull request)

**Comment by kspearrin** [comment]
Can you comment out all uncompilable code so that the solution will build? I do not want to merge code that doesn't compile.

**Comment by amoraitis** [comment]
Ok, @kspearrin I will work on it further more! Just post a message with the forked repo in gitter to find someone to help with this!

**Comment by amoraitis** [comment]
@kspearrin ok, DONE

**Pull request closed without merging**